js/src/sheet.js

```diff
@@ -317,6 +318,8 @@ var SheetView = widgets.DOMWidgetView.extend({
             data: this._get_cell_data(),
             rowHeaders: true,
             colHeaders: true,
+            renderAllRows: true,
+            viewportColumnRenderingOffset: 100,
```

Comment:
Could you add comments to this, i.e. explain why it is needed, maybe link to the PR, and/or maybe link to some issue on handsontable (I guess https://github.com/handsontable/handsontable/issues/3570 ) or include more info on the PR.

Reply:
I added comments in line.   
But it is not an handsontable bug. I'd say it looks more like an ipywidget bug. At least there is a miscommunication issue.   
If handsontable virtual rendering is on, the ipywidget messaging system does not succeed in passing the information that new rows/cols should be rendered after scrolling.  
To avoid that, at least temporarily, and because ipysheet are not destined to manipulate very large tables (for that ipyaggrid is a better option it seems to me), disabling virtual rendering altogether for rows and columns is an easy fix.

To test this and the other bug above, I have added a notebooks folder that contain demo notebooks - convenient to test/debug